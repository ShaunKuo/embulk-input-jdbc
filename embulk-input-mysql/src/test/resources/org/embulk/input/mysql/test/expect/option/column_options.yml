table: test3
order_by: 'id'
column_options:
  datetime1: {type: string, timestamp_format: '%Y/%m/%d %H:%M:%S', timezone: "+0000"}
  datetime2: {type: string, timestamp_format: '%Y-%m-%d %H:%M:%S', timezone: "+0900"}
