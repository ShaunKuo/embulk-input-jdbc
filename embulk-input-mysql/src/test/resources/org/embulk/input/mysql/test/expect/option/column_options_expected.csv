1,,
2,2019/01/02 12:34:56,2019-01-02 21:34:56
3,2018/12/31 23:59:59,2019-01-01 08:59:59
